buildscripts/docker/Dockerfile

```diff
@@ -6,5 +6,12 @@ FROM ubuntu:16.04
 
 COPY maya /usr/local/bin
 
-ENTRYPOINT ["maya"]
+ARG VERSION
+ARG BUILD_DATE
+
+RUN echo $BUILD_DATE
```

Comment:
Why is this echo required?

Reply:
Oh! Sorry for that. I was just testing stuff and it got in. Will remove it